lib/components/molecules/ArticleThumbnail/ArticleThumbnail.mock.js

```diff
@@ -0,0 +1,29 @@
+const defaultImage = {
+  src: '/images/dummies/320.png',
+  srcset:
+    '/images/dummies/1280.png 1024w, /images/dummies/640.png 640w, /images/dummies/320.png 320w',
```

Comment:
Can you use different image for different examples. Since we are using the same image everywhere its doesn't get requested again. Use images from below CDN, They are responsive.

https://placekitten.com/

Reply:
done